helpers/notifications.js

```diff
@@ -1,17 +1,18 @@
 const { ObjectId } = require("mongodb");
 const getMongoConnection = require("./mongoConnect");
 const { Expo } = require("expo-server-sdk");
-const cron = require("node-cron");
+const getToken = require("../middleware/verifyAndAttachJWTData");
 
 const attachPushToken = async (req, res, next) => {
   try {
     const decodedTokenData = req.body.userID;
     const pushToken = req.body.expoPushToken;
     const mongoConnection = await getMongoConnection();
+    const userToken = await getToken(req);
     const userRecord = await mongoConnection
       .db("push")
       .collection("users")
-      .updateOne({ _id: ObjectId(decodedTokenData) }, { $set: { expoPushToken: pushToken } });
+      .updateOne({ userToken: ObjectId(userToken) }, { $set: { expoPushToken: pushToken } });
```

Comment:
@anandangalig I wasn't sure what the key would be for this update lookup

Reply:
I am not sure if verifyAndAttachJWTData needs to be imported here like this. It's a middleware that verifies the JWT and passes the deciphered JWT data to the next function. 

Since, my PR added this middleware to app.post("/run-notifications", **verifyAndAttachJWTData**, scheduleNotifications);, all you need is to pull it out of the **req** object in here:

const decodedTokenData = req.tokenData;
decodedTokenData will have several user related info. I am hoping one of them is _id. And you will use that in your updatedOne call to target that specific user record.